### ISSUE TITLE
Various notebook links lead to 403 or 404 pages

### ISSUE THREAD

**Issue by nmb-paperspace** [gradient-ai/Graphcore-Tensorflow2#15]
From the Gradient interface they result in, e.g.,

![Screen Shot 2022-12-19 at 5 40 10 PM](https://user-images.githubusercontent.com/74800360/208561177-e74018d1-81c4-43e2-81a9-3d4a7293d1f9.png)

`learning-TensorFlow2-on-IPU/keras/demo.ipynb`:

“TensorFlow 2 PopDist example” link gives 404  
"the TensorFlow 1 pipelining tutorial" link gives 404

`learning-TensorFlow2-on-IPU/mnist/mnist.ipynb`

x2 "TensorFlow 2 Keras tutorial" links are 404, 403

**Comment by payoto** [comment]
Fixed by #15
